integration/kubernetes/helm-chart/alluxio/Chart.yaml

```diff
@@ -21,3 +21,6 @@ maintainers:
   email: jiacheng@alluxio.com
 - name: Yang Che
   email: cheyang@163.com
+- name: Xiang chen
+  email: cdmikechen@hotmail.com
```

Comment:
This list is the major code maintainers. We have the commitment on maintaining the helm chart and resolving issues. If you volunteer to be one, you are more than welcome! But you don't have to put your name here just because you submitted a bug fix :)

Reply:
@jiacheliu3  OK~ I‘ll remove this